portal-impl/test/unit/com/liferay/portal/kernel/flash/FlashMagicBytesUtilTest.java

```diff
@@ -16,16 +16,8 @@
 
 import com.liferay.portal.test.rule.LiferayUnitTestRule;
 
-import java.io.ByteArrayInputStream;
```

Comment:
@jeffwu0724 The FlashMagicBytesUtil is removed, we do not need the FlashMagicBytesUtilTest at all, please remove it.